Add readme detail and add a new page (#17)
Rurusetto/rurusetto: README.md

 A wiki that contain all osu! rulesets

## Domain

- Dev build (Some function in dark mode like color invert will not work.) : https://dev.rulesets.info/
- Official build : https://rulesets.info/
- Directly to a ruleset page : https://[rulesetname].rulesets.info/

## Development Status

A website structure is almost finish. Current adding more ruleset to website.

## Developing Rūrusetto codebase

> *If you don't want to developing a codebase but you want to contribute a wiki you can skip to [contributing](#contributing) part.*

Reviewer: ```suggestion
+ If you are only interested in contributing to wiki content, instead of the underlying backend, please skip to [contributing](#contributing). 
```

Please make sure you have the following prerequisites:

- [Hugo Framework](https://gohugo.io/)
- Text IDE. We recommend IDE with intelligent code completion and syntax highlighting if you work with a codebase. Our recommendation is [Visual Studio Code](https://code.visualstudio.com/)

> 

Reviewer: Might be worth adding Git into this list.

 A wiki that contain all osu! rulesets

## Domain

> - Dev build (Some function in dark mode like color invert will not work.) : https://dev.rulesets.info/

Reviewer 2: This should be:
Dev build (Some functions in dark mode like color invertion might not work.) : https://dev.rulesets.info/

> - Official build : https://rulesets.info/

Reviewer 2: Official Build: https://rulesets.info/

Is better the way I see it

Reviewer: I don't think this needs capitalization like you suggest. This should follow standard casing as it isn't a proper noun.

Author: So I'm not change it.

- Directly to a ruleset page : https://[rulesetname].rulesets.info/

## Development Status

> A website structure is almost finish. Current adding more ruleset to website.

Reviewer 2: ```suggestion
The website structure is almost finished. Currently adding more rulesets pages to the website and improving on the pages currently in place.
```

## Developing Rūrusetto codebase

> *If you are only interested in contributing to wiki content, instead of the underlying backend, please skip to [contributing](#contributing).*

Reviewer 2: ```suggestion
*If you are only interested in contributing to the wiki content, instead of the underlying backend, please skip to [contributing](#contributing).*
```

Please make sure you have the following prerequisites:

- [Hugo Framework](https://gohugo.io/)
- Text IDE. We recommend IDE with intelligent code completion and syntax highlighting if you work with a codebase. Our recommendation is [Visual Studio Code](https://code.visualstudio.com/)
- [Git](https://git-scm.com/)

### Downloading the source code

Clone the repository:

```shell
git clone https://github.com/ppy/osu
cd osu
```

To update the source code to the latest commit, run the following command inside the Rūrusetto directory:

```shell
git pull
```

### Start a development server

Run this command inside the Rūrusetto directory, this will enable a development in http://localhost:1313/ and this support fast render and reload:

```shell
hugo server
```

### Build this website

> Run this command inside the Rūrusetto directory, complete build will available in `public` directory inside source code directory.

Reviewer 2: ```suggestion
Run this command inside the Rūrusetto directory, a complete build will be available in `public` directory inside source code directory.
```

```shell
hugo
```

> 

Reviewer 2: ```suggestion
## Developing Rūrusetto codebase

*If you are only interested in contributing to the wiki content, instead of the underlying backend, please skip to [contributing](#contributing).*

Please make sure you have the following prerequisites:

- [Hugo Framework](https://gohugo.io/)
- Text IDE. We recommend IDE with intelligent code completion and syntax highlighting if you work with a codebase. Our recommendation is [Visual Studio Code](https://code.visualstudio.com/)
- [Git](https://git-scm.com/)

### Downloading the source code

Clone the repository:

```shell
git clone https://github.com/ppy/osu
cd osu
```

To update the source code to the latest commit, run the following command inside the Rūrusetto directory:

```shell
git pull
```

### Start a development server

Run this command inside the Rūrusetto directory, this will enable a development in http://localhost:1313/ and this support fast render and reload:

```shell
hugo server
```

### Build this website

Run this command inside the Rūrusetto directory, complete build will be available in `public` directory inside source code directory.

```shell
hugo
```

```
